hyftest.js

```diff
@@ -0,0 +1,60 @@
+// 1  Programming Basics
+const movies = ["The Shawshank Redemption",
+"The Godfather",
+"The Dark Knight",
+"12 Angry Men",
+"The Lord Of The Rings",
+"Pulp Fiction",
+"The Good, the Bad and the Ugly",
+"Forrest Gump"];
+
+//1. Log out each movie title;
+console.log(movies)
+
+//2. Log out the number of words in each movie title.
+movies.forEach(function(movies) {
+console.log(`Numbers of words in the ${movies}" is ${movies.split(" ").length}.`);
+   });
```

Comment:
Again, you did not necessary use a `for` loop. However, you guys have been taught during the class the different ways of iterating, may it be a `for` loop, `forEach`, or a `for(let _ of _)`. Thus I understand why you might have wanted to solve this exercise like this. If you do move on to university though, there the professors might be more strict on following the exercise. 😉 

Also, using the tempate literal for the output string is always better than string addition!